R-package/NEWS.md

```diff
@@ -0,0 +1,32 @@
+
+
```

Comment:
Please remove 2 blank lines at the top of the file.

Reply:
fixed and pushed changes.